src/components/CellLineTable/NormalTableColumns.tsx

```diff
@@ -16,17 +16,26 @@ import {
 
 const { lastColumn } = require("../../style/table.module.css");
 
+/* case insensitive string comparison */
+const compareStrings = (a = "", b = "") =>
```

Comment:
nit, but could call this `caseInsenstiveStringCompare` and then you don't need the comment